packages/connect/src/device/Device.ts

```diff
@@ -793,22 +797,24 @@ export class Device extends TypedEmitter<DeviceEvents> {
 
         // handle rejection of call by a counterfeit device. If unhandled, it crashes device initialization,
         // so device can't be used, but it's preferable to display proper message about counterfeit device
-        const getFirmwareHashOptional = async () => {
-            try {
-                return await this.getCommands().typedCall('GetFirmwareHash', 'FirmwareHash', {
-                    challenge,
-                });
-            } catch {
-                return null;
```

Comment:
at this point I need to return from `checkFirmwareHash`, and return the catch(err). Can't do that when the API call is wrapped in `getFirmwareHashOptional`, so I removed that fn.
It was there because you cannot define `const` in `try` block and then access it outside of it.
But the rest of the function is safe, so I believe it can all be in the try block.